Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RHOMBUS.MSG

```diff
@@ -116,28 +116,21 @@
 # Rhombus Tell-Me-Abouts
 #
 
-
 {1000}{}{Vree}
 {1001}{}{Rhombus|Head Paladin}
 {1002}{}{Talus|Talius}
 {1003}{}{Elder}
 {1004}{}{Scribes|Scribe}
 {1005}{}{Knights|Knight}
 {1006}{}{Hub}
-{1007}{}{Disks|Holodisk|Holodisks|Disk}
+{1007}{}{Disks|Holodisk|Holodisks|Disk|Master}
 {1008}{}{Army|Strange Army|Mutant Army}
 {1009}{}{Boneyard|LA|Los Angeles|Angels Boneyard}
 {1010}{}{Paladins|Paladin}
-{1011}{}{Master}
-{1012}{}{War}
-{1013}{}{Exodus}
+{1012}{}{War|Exodus|Merchants|Merchant|Brotherhood|BoS|Brotherhood of Steel|Initiates|Initiate}
 {1014}{}{Strange Army}

```

Comment:
Missing line 1011, 1013, 1015. Bad.

Reply:
He should now have precisely 15 Tell-Me-Abouts. {1000} to {1014}, {1100} to {1114}.